tests/e2e/test_vehicle.py

```diff
@@ -125,16 +125,17 @@ def test_set_charge_limit(ford_car):
 
 
 def test_batch_success(chevy_volt):
-    batch = chevy_volt.batch(["/odometer", "/location"])
+    batch = chevy_volt.batch(["/odometer", "/location", "/charge/limit"])
```

Comment:
to be safe, can you also add one of engin_oil, tire_pressure, or battery_Capacity in this test ?

Reply:
Sure thing.

Reply:
Good idea on the tire_pressure check.

Our path is `/tires/pressure` but our function call is `tire_pressure`.

Reply:
Ready for review.